cloudinary-core/src/main/java/com/cloudinary/Uploader.java

```diff
@@ -282,13 +292,28 @@ public Map generateSprite(String tag, Map options) throws IOException {
             transformation.fetchFormat(format);
         }
         params.put("transformation", transformation.generate());
-        params.put("tag", tag);
+        params.put("tag", options.get("tag"));
+        if (options.containsKey("urls")) {
+            params.put("urls", Arrays.asList((String[]) options.get("urls")));
+        }
         params.put("notification_url", (String) options.get("notification_url"));
         params.put("async", ObjectUtils.asBoolean(options.get("async"), false).toString());
+        params.put("mode", options.get("mode"));
+
         return callApi("sprite", params, options, null);
     }
 
+    public Map multi(String[] urls, Map options) throws IOException {
+        options.put("urls", urls);
```

Comment:
Don't you check for array size >=1

Reply:
We don't run validations at all, we can start if you want but that's not even the tip of the iceberg...

Reply:
I know we usually don't, and I wouldn't want to retroactively cover all, but in new code, if a validation is calling out + it's easy to validate + it's something that has very low chances of being changed in the BE (we can assume it won't expect an empty array), then why not?